cli/samples/default-config.yaml

```diff
@@ -21,18 +21,16 @@ vpcConfig:
   authIP: "10.10.10.10"
 clustersConfig:   
 - clusterName: "gke-central"
-  projectId: "my-project"
   numNodes: 3 
   clusterType: "public"
   machineType: "e2-standard-4"
   region: "us-central1" 
   zone: "us-central1-b"
   subnetName: "us-central1"
 - clusterName: "gke-east"
-  projectId: "my-project"
```

Comment:
Why did you remove these?

Reply:
Good catch. lmk fix

Reply:
Oh wait, I removed those because they are not used. All cluster projects are derived from the base vars "clustersProjectId"

Reply:
gotcha, then should we remove the cluster config field `projectId` ? that's still in config.go

Reply:
Done, thanks!